build.rs

```diff
@@ -11,7 +11,10 @@ fn sdk_path(target: &str) -> Result<String, std::io::Error> {
 
     let sdk = if target.contains("apple-darwin") {
         "macosx"
-    } else if target == "x86_64-apple-ios" || target == "i386-apple-ios" {
+    } else if target == "x86_64-apple-ios"
+        || target == "i386-apple-ios"
+        || target == "aarch64-apple-ios-sim"
```

Comment:
Let's add this to the [CI build list](https://github.com/RustAudio/coreaudio-sys/blob/5837e3a1ce84f747ec96be956ca0b6d85d7b1736/.github/workflows/coreaudio-sys.yml#L27).